src/components/LayoutTree.tsx

```diff
@@ -540,50 +539,18 @@ const TreeNode = ({
         </div>
       </FadeTransition>
 
-      {/* DropEnd (cliff) */}
       {dragInProgress &&
-        cliff < 0 &&
         // do not render hidden cliffs
         // rough autofocus estimate
-        autofocusDepth - depth < 2 &&
-        Array(-cliff)
-          .fill(0)
-          .map((x, i) => {
-            const pathEnd = -(cliff + i) < path.length ? (path.slice(0, cliff + i) as Path) : HOME_PATH
-            const cliffDepth = unroot(pathEnd).length
-
-            // After table col2, shift the DropEnd left by the width of col1.
-            // This correctly positions the drop target for dropping after the table view.
-            // Otherwise it would be too far to the right.
-            const dropEndMarginLeft =
-              isTableCol2 && cliffDepth - depth < 0 ? treeThoughtsPositioned[index - 1].width || 0 : 0
-
-            return (
-              <div
-                key={'DropEnd-' + head(pathEnd)}
-                className={css({
-                  position: 'relative',
-                  top: '-0.2em',
-                  transition: `left {durations.fast} ease-out`,
-                  zIndex: 'subthoughtsDropEnd',
-                })}
-                style={{
-                  left: `calc(${cliffDepth - depth}em - ${dropEndMarginLeft}px + ${isTouch ? -1 : 1}px)`,
-                }}
-              >
-                <DropEnd
-                  depth={pathEnd.length}
-                  path={pathEnd}
-                  cliff={cliff}
-                  isLastVisible={isLastVisible}
-                  // Extend the click area of the drop target when there is nothing below.
-                  // The last visible drop-end will always be a dimmed thought at distance 1 (an uncle).
-                  // Dimmed thoughts at distance 0 should not be extended, as they are dimmed siblings and sibling descendants that have thoughts below
-                  // last={!nextChildId}
-                />
-              </div>
-            )
-          })}
+        autofocusDepth - depth < 2 && (
+          <DropCliff
+            cliff={cliff}
+            depth={depth}
+            path={path}
+            isTableCol2={isTableCol2}
+            prevWidth={treeThoughtsPositioned[index - 1]?.width}
```

Comment:
but these 2 maybe suck

I also initially did a 
`marginLeft={isTableCol2 ? treeThou...` and applied it inside the `DropCliff` with a condition

but it kinda felt like more vague, though half as verbose

Reply:
We could pass a raw CSS object to the `DropCliff` which has the `left` margin calculated, but I believe it makes more sense to let the `DropCliff` calculate the styling on its own -- especially because the component is only used in one place and doesn't need to let the parent component determine this.

I agree that it doesn't feel ideal to pass values such as `isTableCol2` and `prevWidth`, but it is the right choose in this case in my opinion.